node-src/lib/getOptions.ts

```diff
@@ -77,6 +85,7 @@ export default function getOptions({ argv, env, flags, log, packageJson }: Conte
     patchHeadRef,
     patchBaseRef,
   };
+  const options = { ...optionsFromFlags, ...configuration, ...extraOptions };
```

Comment:
I agree, but unfortunately it's not as simple as that because the `optionsFromFlags` set a bunch of defaults, and it's not totally clear how we should handle it in all cases. I'll work through it with @ghengeveld tonight.